damus/Models/TranslationService.swift

```diff
@@ -38,7 +38,13 @@ enum TranslationService: String, CaseIterable, Identifiable, StringCodable {
     var model: Model {
         switch self {
         case .none:
-            return .init(tag: self.rawValue, displayName: NSLocalizedString("none_translation_service", value: "None", comment: "Dropdown option for selecting no translation service."))
+            let displayName: String
+            if TranslationService.isAppleTranslationPopoverSupported {
```

Comment:
I'd like to solicit feedback here. Right now, if you have iOS 17.4+, there's no way to turn off translations. It's just using Apple by default, but requires an explicit tap on the Translate Note button.

Alternatively, I could make `None` and `Apple` separate options, but it feels like we have an opportunity to be more opinionated here to just say everyone needs translations.

Reply:
I think we need to consider the privacy aspect, given that some of those translate requests might be sent to Apple servers.

In my opinion, we should be opinionated and place the button "Translate note" by default, and then add a confirmation dialog explaining the user the text _might_ be sent to Apple servers, with an option to dismiss that dialog forever if the user chooses to do so.

For the iOS 18 feature where translations can be made automatically, I believe that should be opt-in.

Reply:
> I think we need to consider the privacy aspect, given that some of those translate requests might be sent to Apple servers.
> 
> In my opinion, we should be opinionated and place the button "Translate note" by default, and then add a confirmation dialog explaining the user the text _might_ be sent to Apple servers, with an option to dismiss that dialog forever if the user chooses to do so.

![Simulator Screenshot - iPhone 16 Pro - 2024-09-22 at 00 44 41](https://github.com/user-attachments/assets/0a14d999-34c6-4050-be58-4822f5f4cb2e)

It turns out that Apple shows exactly that confirmation dialog on the first attempt, so I think we're OK there. Agreed about the automatic iOS 18 translations feature, though.

Reply:
Thank you @tyiu, I will take a second look at this PR soon with this in mind!